### ISSUE TITLE
Direct Message

### ISSUE THREAD

**Issue by Long1351** [DarkGuy10/BotClient#42]
I can't get direct messages from a user

**Comment by aiko-chan-ai** [comment]
> I can't get direct messages from a user

maybe the client doesn't open the partial to receive the message

**Comment by DarkGuy10** [comment]
@aiko-chan-ai correct, the client doesn't use partials, I tried checking if turning on channel partials directly solves the issue but it doesn't. The IPC / client events need to be slightly tweaked
